docs/evidence/change-aware-ci-proof-of-life.md

```
@@ -0,0 +1,295 @@
+---
+
+### Verify branch-protection gates
+
+```bash
+gh api "repos/OmniNode-ai/omnibase_core/branches/main/protection/required_status_checks/contexts" \
+  --jq '.[]' | sort
+```
+
+Expected output (exactly these three, no extras):
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)